### PR TITLE
[KEYBOARD] Switch to QWERTZ for Croatian keyboard layout

### PR DESCRIPTION
**Pull request by Andrej123456789**
## Purpose

Croatian keyboards are QWERTZ.

JIRA issue: unknown

## Proposed changes

Update kbdcr.c
## TODO
Nothing, I builded and tested in virtual machine and works.

### CODE REVIEW

File: dll/keyboard/kbdcr/kbdcr.c [hunk 1]
```diff
@@ -41,7 +41,7 @@ ROSDATA USHORT scancode_to_vk[] = {
   /* - 0f - */
   /* First Letters Row */
   VK_TAB,       'Q',          'W',          'E',
-  'R',          'T',          'Y',          'U',
+  'R',          'Z',          'Y',          'U',
```
Comment (HBelusca) [inline dll/keyboard/kbdcr/kbdcr.c:44]: Shouldn't it be
```suggestion
  'R',          'T',          'Z',          'U',
```
?

Comment (Andrej123456789) [reply dll/keyboard/kbdcr/kbdcr.c:44]: Always I did something wrong; yeah I did mistake and I will fix it

**Comment by HBelusca** [comment]
Since you're looking at the keyboard stuff, could you tell me whether the following keys do work:
- `?`, `'` and `¨` (they are near the `0` key in first row) ;
- `*`, `+` and `¸` (they are near the previous key in first row) ;
- `\` (in same key as `Q`) and `|` (in same key as `W`) in second row;
- `@` (in same key as `V`), `{` , `}` and `§` (in same keys as `B`, `N` and `M`) ;
- and the special characters `š`, `đ`, `č`, `ć` and `ž`

**Comment by Andrej123456789** [comment]
Everything works as normal

**Comment by HBelusca** [comment]
Hvala!

**Pull request merged**